1-js/05-data-types/03-string/article.md

```diff
@@ -635,42 +635,42 @@ alert( 'S\u0323\u0307' ); // Ṩ, S + dot below + dot above
 alert( 'S\u0307\u0323' == 'S\u0323\u0307' ); // false
 ```
 
-To solve it, there exists a "unicode normalization" algorithm that brings each string to the single "normal" form.
+To solve this, there exists a "unicode normalization" algorithm that brings each string to the single "normal" form.
 
 It is implemented by [str.normalize()](mdn:js/String/normalize).
 
 ```js run
 alert( "S\u0307\u0323".normalize() == "S\u0323\u0307".normalize() ); // true
 ```
 
-It's funny that in our situation `normalize()` actually brings a sequence of 3 characters to one: `\u1e68` (S with two dots).
+It's funny that in our situation `normalize()` actually brings together a sequence of 3 characters to one: `\u1e68` (S with two dots).
 
 ```js run
 alert( "S\u0307\u0323".normalize().length ); // 1
 
 alert( "S\u0307\u0323".normalize() == "\u1e68" ); // true
 ```
 
-In real, that is not always so. The reason is that symbol `Ṩ` is "common enough", so UTF-16 creators included it into the main table and gave it the code.
+In reality, this is not always the case. The reason being that the symbol `Ṩ` is "common enough", so UTF-16 creators included it in the main table and gave it to the code.
```

Comment:
"gave it to the code" - typo?

Reply:
This is a difficult statement to interpret (from my beginner point-of-view).

Are you saying here that:

"S\u0307\u0323".normalize() == "\u1e68" // true 

because UTF-16 includes the `Ṩ` symbol in the spec, but not all symbols are included in this spec., so .normalize will not always translate to 'true'?

If I misunderstand this part, then I will leave your original wording so that it is technically correct.

Reply:
I mean that "S\u0307\u0323" gets normalized to a single character, because the symbol "s with two dots" is very common and has a separate code.

Reply:
Hmm I think the wording should stay then, and just remove 'to'.